### ISSUE TITLE
Improved instructions to run systemd enabled app container, and instructions for Fedora

### ISSUE THREAD

**Issue by KamranAzeem** [CentOS/sig-cloud-instance-images#152]
Hi,
Thanks for your great work!
I came across the instructions to run "systemd enabled app container" on your DockerHub page (https://hub.docker.com/_/centos) . 

> Running a systemd enabled app container
> In order to run a container with systemd, you will need to mount the cgroups volumes from the host. Below is an example command that will run the systemd enabled httpd container created earlier.
> 
> $ docker run -ti -v /sys/fs/cgroup:/sys/fs/cgroup:ro -p 80:80 local/c7-systemd-httpd
> This container is running with systemd in a limited context, with the cgroups filesystem mounted. 
> There have been reports that if you're using an Ubuntu host, you will need to add -v /tmp/$(mktemp -d):/run in addition to the cgroups mount.

First, I would like to confirm that `/run` needs to be mounted when you run this container on Fedora too. So this is true for Fedora and Ubuntu.

Furthermore, this extra mount required for Fedora and Ubuntu can actually be mounted in an improved way by replacing `/tmp/$(mktemp -d):/run`  with `--tmpfs /run` . So the end result looks like this:
```
$ docker run -it -v /sys/fs/cgroup:/sys/fs/cgroup:ro --tmpfs /run -p 80:80 local/c7-systemd-httpd
```

Also, if one wants to run this (systemd enabled container) through docker-compose, the following should be used:
```
version: "3"
services:
  myapp:
    build: .
    ports:
      - "80:80"
    volumes:
      - /sys/fs/cgroup:/sys/fs/cgroup:ro
    tmpfs:
      - /run  
```
I hope you add that to the README. If you want me to do it, perhaps I can create a PR?

Using RedHat/CENTOS/Fedora for last ~20 years! Cheers!

**Comment by qhaas** [comment]
Indeed, I applaud the work of the CentOS/RH teams, especially [Dan Walsh of Red Hat](https://lwn.net/Articles/676831/) for going down the rabbit hole years ago and getting us to the point of having rootless/non-privileged containers running systemd **with SELinux/AppArmor enforcing** on the host.

I have similar findings to report [for Ubuntu 18.04 and CentOS7 hosts and Docker 19.03](https://forums.docker.com/t/systemctl-status-is-not-working-in-my-docker-container/9075/19) regarding the need for `/run`.  It seems earlier versions of Docker (1.13 that ships with EL7) don't need '/run', but modern versions of docker running on EL7 hosts do.
